gyucheol/5일차/음양_더하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(absolutes, signs) {
+    let sum = 0;
+    for( let i = 0; i < absolutes.length; i++){
+        sum += signs[i] ? absolutes[i] : -absolutes[i];
+            return sum;
+        }
+    }
```

Comment:
질 풀었습니다~
컨벤션 주의해 주세요 ㅎㅎ
```suggestion
    let sum = 0;
    
    for (let i = 0; i < absolutes.length; i++) {
        sum += signs[i] ? absolutes[i] : -absolutes[i];
            return sum;
        }
    }
    
    // reduce를 활용한 한줄 풀이
    return absolutes.reduce((acc, el, i) => (
        acc + (el * (signs[i] ? 1 : -1))
    ), 0)
```